jupyter-base-notebook.yaml

```diff
@@ -86,6 +91,20 @@ test:
         expected_output: |
           extension was successfully linked
           extension was successfully loaded
+    - name: Run and test jupyterhub-singleuser
+      uses: test/daemon-check-output
+      with:
+        setup: |
+          jupyterhub >/dev/null 2>&1 &
+          sleep 3
+        start: jupyterhub-singleuser
+        timeout: 30
+        expected_output: |
+          nbclassic | extension was successfully loaded.
+          notebook | extension was successfully loaded.
+          Serving notebooks from local directory: /home/build
+          Jupyter Server 2.15.0 is running at:
```

Comment:
we can consider using a subset of words here else we'll run into error when jupyter version moves up. 💭